paper/paper.md

```diff
@@ -0,0 +1,109 @@
+   index: 2
+ - name: Massachusetts Institute of Technology
+   index: 3
+ - name: Australian National University
+   index: 4
+ - name: Australian Research Council Centre for Climate Extremes
+   index: 5
+date: 15 March 2023
+bibliography: paper.bib
+---
+
+# Summary
+
+``OceanBioME.jl`` is a flexible modelling environment written in Julia [@julia] for modelling the coupled interactions between ocean biogeochemistry, carbonate chemistry, and physics.
+``OceanBioME.jl`` can be used as a stand-alone box model, or integrated into ``Oceananigans.jl`` [@Oceananigans] simulations of ocean-flavoured fluid dynamics in one-, two-, or three-dimensions.
+As a result, ``OceanBioME.jl`` and ``Oceananigans.jl`` can be used to simulate the biogeochemical response across an enormous range of scales: from surface boundary layer turbulence at the meter scale to eddying global ocean simulations at the planetary scale, and on computational systems ranging from laptops to supercomputers.
+An example of a problem involving small-scale flow features is shown in \autoref{fig1}.
+``OceanBioME.jl`` leverages Julia's multiple dispatch and effective inline capabilities to fuse its computations directly into existing ``Oceananigans.jl`` kernels, thus maintaining ``Oceananigans.jl``'s bespoke performance, memory- and cost-efficiency on GPUs in ``OceanBioME.jl``-augmented simulations.
+
+![Here we replicate the Eady problem where a background buoyancy gradient and corresponding thermal wind generate a sub-mesoscale eddy, roughly following the setup of Taylor (2016).
+To this physical setup, we added a medium complexity (9 tracers) biogeochemical model, some of which are shown above.
+On top of this, we added particles modelling the growth of sugar kelp which are free-floating and advected by the flow, and carbon dioxide exchange from the air.
+Thanks to Julia's speed and efficiency the above model (1 km × 1 km × 100 m with 64 × 64 × 16 grid points) took about 30 minutes of computing time to simulate 10 days of evolution on an Nvidia P100 GPU. Figure made with `Makie.jl` [@makie]. \label{fig1}](eady_example.png)
+
+``OceanBioME.jl`` is built with a highly modular design that allows user control and customization.
+There are three distinct module types implemented in ``OceanBioME.jl``:
+
+- First, tracer-based ecosystem modules are formulated as a set of coupled ordinary differential equations.
+These equations can be solved by ``OceanBioME.jl`` as box models, which is particularly useful for testing.
+The same modules can be integrated by ``Oceananigans.jl`` to provide tracer-based ecosystem models.
+
+- Second, boundary modules contain sets of equations which provide information at the top and bottom of the ocean.
+For example, air-sea gas exchange modules calculate the flux of carbon dioxide and oxygen at the sea surface, while sediment modules calculate fluxes of carbon and oxygen at the seafloor.
+
+- The third module type is "biologically active" particles.
```

Comment:
is the module in question here the `Individuals` (`SLatissima`) as opposed to the tracer-based ecosystems which are in `AdvectedPopulations` (described in the first point of this bullet list) or `Particles`?

In the former case, wouldn't it make sense to rewrite this list by separating out the boundary conditions and highlighting that there are two `model` types (individuals based on lagrangian particle tracking and advected populations based on species transport)

Disregard if I got it wrong...

Reply:
I think this makes sense so will have a go at rewriting that bit now